src/emmy/expression/compile.cljc

```diff
@@ -491,6 +267,31 @@
                  {'up struct/up
                   'down struct/down}}}))
 
+(defn- commafy-arglist
+  "Since JavaScript likes commas between array elements:
+
+   [a [b c [d e] f]] -> \"[a, b, c, d, e, f]\"       if flatten?
+   [a [b c [d e] f]] -> \"[a, [b, c, [d, e], f]]\"   if not flatten?
```

Comment:
did you mean to add a flatten? keyword arg?

Reply:
no, this came out in the wash (fixed doc)